manticore/ethereum/__init__.py

```diff
@@ -336,27 +336,33 @@ def _run_solc(source_file, solc_bin=None, solc_remaps=[]):
             #logger.warning("Unsupported solc version %s", installed_version)
             pass
 
-        #shorten the path size so library placeholders wont fail.
```

Comment:
Can notwe use this path shortening thing in `solidity_create_contract` if the working_directory is not specified?

Reply:
We resolved this on a DM; for others/future ref:

if someone lanuches:
```manticore some_path```
it should launch `solc` with the same path and from the same working dir as the person is on -
unless the person wants it differently.

Previously we shortened the path but we can't do it as we can't know how much can we shorten it as there might be relative imports in contracts :s.